### PR TITLE
Fix recipe error message when no variables are included

### PR DESCRIPTION
**Pull request by rswamina**
# Pull Request checklist

We appreciate your time and effort to improve the tutorial. Please keep in mind that lesson maintainers are volunteers and it may be some time before they can respond to your contribution.

* * *

## Before you start

- [x] Read [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md).
- [x] Create an [issue](https://github.com/ESMValGroup/tutorial/issues) to discuss your idea. This allows your contributions to be incorporated into the tutorial.

## Tasks

- [x] Give this pull request a descriptive title.
- [x] If you are contributing to existing lesson materials, please make sure the content conforms to the `Lesson development` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [ ] If you are making a new episode, please make sure the content conforms to the `Lesson organization` and `Lesson formatting` sections in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) and does not contain any spelling or grammatical errors.
- [x] Preferably Codacy checks pass. Status can be seen below your pull request. If there is an error, click the link to find out why.
- [x] Preview changes on your machine before pushing them to GitHub by running `make serve`, alternatively `make docker-serve`. Please see the `Previewing your changes locally` section in [CONTRIBUTING.md](https://github.com/ESMValGroup/tutorial/blob/master/CONTRIBUTING.md) for installation instructions.
- [x] All code instructions have been tested.

If you need help with any of the tasks above, please do not hesitate to ask by commenting in the issue or pull request.

* * *

Closes [229](https://github.com/ESMValGroup/ESMValTool_Tutorial/issues/229)

### CODE REVIEW

**Comment by rswamina** [comment]
@remi-kazeroni  and @SarahAlidoost  - We were alerted to a couple of issues from participants in today's tutorial session. I have made changes to reflect the actual message in the log file when a recipe is run without variables. Please take a look. I also removed a reference to a local data directory since we don't use those for the most part.

File: _episodes/06-preprocessor.md [hunk 1]
```diff
@@ -219,18 +222,23 @@ previous episodes. The data files are stored in `~/esmvaltool_tutorial/data`.
...
+We start with the BCC-ESM1 dataset and add a datasets section to the recipe,
+listing a single dataset, as shown below. Note that key fields such 
+as `mip` or `start_year` are included in the `datasets` section here but are part 
+of the `diagnostic` section in the recipe example seen in [Running your first recipe]({{ page.root }}{% link _episodes/04-recipe.md %}).
 
 ```yaml
 datasets:
-  - {dataset: BCC-ESM1, project: CMIP6, mip: Amon, exp: historical, ensemble: r1i1p1f1, grid: gn, start_year: 1850, end_year: 2014}
+  - {dataset: BCC-ESM1, project: CMIP6, mip: Amon, exp: historical, 
+ensemble: r1i1p1f1, grid: gn, start_year: 1850, end_year: 2014}
```
[the remark this answers is not in the document]
Comment (SarahAlidoost) [reply _episodes/06-preprocessor.md:233]: It seems that the line "dataset" was split into two lines. I fixed it above.

**Review by SarahAlidoost — APPROVED** [review]
@rswamina thanks, looks good to me. I left a minor comment.

**Comment by rswamina** [comment]
@remi-kazeroni -- If you think all concerns raised at the tutorial with regards to this issue are satsified with these edits, then we can merge the changes so participants have an updated version of this lesson soon. If you want me to make further edits, please let me know as well.

**Review by remi-kazeroni — APPROVED** [review]
Thanks for the changes @rswamina! I think all concerns raised at the last tutorial have been addressed.

**Pull request merged**